### ISSUE TITLE
Use numbers instead of percentages

### ISSUE THREAD

**Issue by khorammfar** [leongersen/noUiSlider#1210]
I need to give the range of numbers to the slider myself instead of using %.

For example instead of writing this

`range: {'min': 10, 'max': 2000, "2%": 200, "20%": 600}`

use this:
```
{
  min: 100,
  max: 20000,
  range: [ [10, 1], [100, 10], [1000, 100] ]
}

```
For example, when the suffering number reaches 100, take ten to ten steps.
When it reaches 1000, the steps will be 100 steps

Can such steps be specified?

**Comment by leongersen** [comment]
Yes, you can use steps like that. You'll just need to specify the percentages to position the starting locations for those steps on the slider.

Have a look at:
https://refreshless.com/nouislider/slider-values/#section-non-linear-step

**Comment by khorammfar** [comment]
It does not work very interestingly on some sufferings
Take a look at this link
`https://socialstore.io/telegram-views/`
I implemented the same with percentage
ranges for every product are different
